src/calculateRentalCost.js

```diff
@@ -1,10 +1,20 @@
 /**
  * @param {number} days
- *
  * @return {number}
  */
 function calculateRentalCost(days) {
-  // write code here
+  const dailyRate = 40; // Вартість оренди за день
```

Comment:
```suggestion
  const dailyRate = 40;
```
remove all comments from the file. It is not a good practice to comment the code